src/inc/win64unwind.h

```diff
@@ -0,0 +1,117 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#ifndef _WIN64UNWIND_H_
+#define _WIN64UNWIND_H_
+
```

Comment:
This header file should also be under #if defined(_TARGET_AMD64_). Win64 would also apply to ARM64.